spec/config_hub/client_spec.rb

```diff
@@ -62,6 +62,14 @@
       end
     end
 
+    describe '#to_h' do
+      it 'returns a hash of the keys to values from the properties' do
+        expect(subject.to_h).to include(
+          'salesforce.web_host' => 'http://scheduling.example.com',
+          'thiskeyisnil' => nil
+        )
```

Comment:
Nit: Can we be more precise than `include`? Are the `Hash` contents unpredictable?